core/src/main/scala/org/bitcoins/core/psbt/PSBTMap.scala

```diff
@@ -621,6 +621,49 @@ case class InputPSBTMap(elements: Vector[InputPSBTRecord])
 object InputPSBTMap extends PSBTMapFactory[InputPSBTRecord, InputPSBTMap] {
   import org.bitcoins.core.psbt.InputPSBTRecord._
 
+  def fromWitness(witness: ScriptWitness): InputPSBTMap = {
+    witness match {
+      case p2wpkh: P2WPKHWitnessV0 =>
+        val sig = PartialSignature(p2wpkh.pubKey, p2wpkh.signature)
```

Comment:
you need to check that there is a signature first, it is possible to have a p2wpkh witness that only has the pubkey ... sadly we don't have separate types for these two things